bundles/binding/org.openhab.binding.zwave/database/lscontrol/e861c.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Product>
+	<Model>E861C</Model>
+	<Label lang="en">Temperature Sensor</Label>
+	<CommandClasses>
+		<Class><id>0x20</id></Class>
+		<Class><id>0x80</id></Class>
+		<Class><id>0x86</id></Class>
+		<Class><id>0x84</id></Class>
+		<Class><id>0x70</id></Class>
+		<Class><id>0x31</id></Class>
+		<Class><id>0x72</id></Class>
+		<Class><id>0x85</id></Class>
+	</CommandClasses>
+	<Configuration>
+		<Parameter>
+			<Index>1</Index>
+			<Type>byte</Type>
+			<Default>0</Default>
+			<Size>1</Size>
+			<Label lang="en">Delta Percentage</Label>
+			<Help lang="en">2 % - 56 % -> Delta</Help>
+		</Parameter>
+		<Parameter>
+			<Index>2</Index>
+			<Type>byte</Type>
+			<Default>0</Default>
+			<Size>1</Size>
+			<Label lang="en">Celsius/Fahrenheit</Label>
+			<Help lang="en">0 - 64 -> Celsius : 64 - 127 -> Fahrenheit</Help>
+		</Parameter>
+		<Parameter>
+			<Index>3</Index>
+			<Type>byte</Type>
+			<Default>0</Default>
+			<Size>1</Size>
+			<Label lang="en">Send Setpoint when WakeUp</Label>
+			<Help lang="en">0 - 64 -> SEND : 64 - 127 -> DO NOT SEND</Help>
+		</Parameter>
+	</Configuration>
+    <Associations>
+    	<Group>
+			<Index>1</Index>
```

Comment:
I guess given this is a status report association it should have <SetToController> set to true?